py/modmath.c

```diff
@@ -171,6 +171,45 @@ MATH_FUN_1(lgamma, lgamma)
 #endif
 //TODO: fsum
 
+static inline mp_float_t get_parsed_float(const mp_obj_t parsed_val, mp_float_t def) {
+    return parsed_val == MP_OBJ_NULL ? def : mp_obj_get_float(parsed_val);
+}
+
+STATIC mp_obj_t mp_math_isclose(size_t n_args, const mp_obj_t *args, mp_map_t *kwargs) {
+    enum { ARG_a, ARG_b, ARG_rel_tol, ARG_abs_tol };
+    static const mp_arg_t allowed_args[] = {
+        {MP_QSTR_a, MP_ARG_REQUIRED | MP_ARG_OBJ},
+        {MP_QSTR_b, MP_ARG_REQUIRED | MP_ARG_OBJ},
+        {MP_QSTR_rel_tol, MP_ARG_KW_ONLY | MP_ARG_OBJ, {.u_obj = MP_OBJ_NULL}},
+        {MP_QSTR_abs_tol, MP_ARG_KW_ONLY | MP_ARG_OBJ, {.u_obj = MP_OBJ_NULL}},
```

Comment:
It is possible to make constant float objects and point to them here, but that'll probably make the code bigger.

Reply:
They won't be static though so allowed_args also won't be static so the end result might be less performant.

Reply:
Since the default value of `abs_tol` is zero it can be `{.u_obj = MP_OBJ_NEW_SMALL_INT(0)}`, and then always use `mp_obj_get_float(...)` to extract it below.  This should save code size.